src/test/java/co/rsk/bitcoinj/script/StandardRedeemScriptParserTest.java

```diff
@@ -71,7 +70,7 @@ public void getSigInsertionIndex_fast_bridge_redeem_script() {
 
         byte[] txSigEncoded = txSig.encodeToBitcoin();
 
-        int sigIndex = parser.getSigInsertionIndex(sigHash, ecKey1);
+        int sigIndex = inputScript.getSigInsertionIndex(sigHash, ecKey1);
```

Comment:
Shouldn't these test cases be removed from this class actually?

Reply:
Tests removed.